js/reducers/user.js

```diff
@@ -39,6 +41,7 @@ export type UserState = LoggedOutUserState | LoggedInUserState
 // initial user state
 const initialUserState: LoggedOutUserState = {
```

Comment:
this would be a `DefaultLoggedOutUserState`

Reply:
fixed